includes/RequestWiki/SpecialRequestWikiQueue.php

```diff
@@ -60,7 +60,8 @@ private function doPagerStuff() {
 					'Unreviewed' => 'inreview',
 					'Approved' => 'approved',
 					'Declined' => 'declined',
-					'On hold' => 'onhold',
+					'On hold (further review)' => 'onhold',
+					'Needs more details' => 'moredetails',
```

Comment:
This option needs to be introduced to RequestWikiRequestViewer, in the `submission-action` key of the form descriptor (https://github.com/miraheze/CreateWiki/blob/master/includes/RequestWiki/RequestWikiRequestViewer.php#L244).

Reply:
Will also need to add some logic in `submitForm` (https://github.com/miraheze/CreateWiki/blob/master/includes/RequestWiki/RequestWikiRequestViewer.php#L342)

Reply:
and in the `WikiRequest` class.